2009-01-10  Aurelien Jarno  <aurel32@debian.org>

	* locales/en_GB: Set first weekday and workday to 1.

diff --git a/localedata/locales/en_GB b/localedata/locales/en_GB
index fe58f1f..d4d5afe 100644
--- a/localedata/locales/en_GB
+++ b/localedata/locales/en_GB
@@ -122,8 +122,8 @@ date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
 week    7;19971201;4
-first_weekday 2
-first_workday 2
+first_weekday 1
+first_workday 1
 END LC_TIME
 
 LC_MESSAGES
